static/js/util/util.js

```diff
@@ -228,40 +228,33 @@ complete profile consists of:
 */
 export function validateProfileComplete(profile) {
   let reqFields = [];
+  let validationMessages = {};
```

Comment:
do you need to build up this object of all the validation messages? in the function body below we return any time we find validation errors relevant to a particular profile tab, so I think for each you should be able to just do something like:

```js
let errors = validateProfile(profile, reqFields, PersonalTab.defaultProps.validationMessages);
```

then we check to see if that's empty, and if not, we return right there. So in any case where we move on to the next tab (professional) the validation messages for the personal tab won't be necessary to check.